.dockerignore

```diff
@@ -1 +1,7 @@
 models/
+docs/
```

Comment:
Remove this, we want to only copy the relevant project files and folders nothing else

Reply:
removed dockerignore